ci/cloudbuild/unit.yaml

```diff
@@ -0,0 +1,27 @@
+steps:
+  # the git steps are required to initialize a clean repo since
+  # Cloud Build does not include the .git directory our scripts
+  # use to orient themselves.
+- name: golang:1.12  # reuse the golang:1.12 image so it's cached for later
```

Comment:
nit: Should we actually be using go1.13?

Reply:
I _think_ go 1.13 has breaking changes to gomod incompatible with our installer last time we checked; our Readme, travis and docker image all use 1.12 currently

Reply:
Fair enough, lets make a directed effort towards moving to it (outside of this PR).